hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestLeaseRecoverer.java

```diff
@@ -14,7 +14,7 @@
  * License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.apache.hadoop.ozone.debug;
+package org.apache.hadoop.ozone;
```

Comment:
I think this should be `package org.apache.hadoop.ozone.admin.om.lease` (to match the code it is testing).